pkg/pillar/cmd/zedmanager/handleclusterapp.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package zedmanager
+
+import "github.com/lf-edge/eve/pkg/pillar/types"
+
+func handleENClusterAppStatusCreate(ctxArg interface{}, key string, configArg interface{}) {
+	log.Functionf("handleENClusterAppStatusCreate(%s)", key)
+	ctx := ctxArg.(*zedmanagerContext)
+	status := configArg.(types.ENClusterAppStatus)
+	handleENClusterAppStatusImpl(ctx, key, &status)
+}
+
+func handleENClusterAppStatusModify(ctxArg interface{}, key string, configArg interface{}, oldConfigArg interface{}) {
+	log.Functionf("handleENClusterAppStatusModify(%s)", key)
+	ctx := ctxArg.(*zedmanagerContext)
+	status := configArg.(types.ENClusterAppStatus)
+	handleENClusterAppStatusImpl(ctx, key, &status)
+}
+
+func handleENClusterAppStatusDelete(ctxArg interface{}, key string, configArg interface{}) {
+	log.Functionf("handleENClusterAppStatusDelete(%s)", key)
+	ctx := ctxArg.(*zedmanagerContext)
+	//status := configArg.(types.ENClusterAppStatus)
+	handleENClusterAppStatusImpl(ctx, key, nil)
+}
+
+func handleENClusterAppStatusImpl(ctx *zedmanagerContext, key string, status *types.ENClusterAppStatus) {
```

Comment:
This function should be updated from our POC branch. Or I can do it as part of my PR for failover handling.

Reply:
Actually I suggest amending this commit with latest handleclusterapp.go and applogs.go from POC branch.
In that way they can be reviewed as a unit and my follow up PRs need not include those files.